src/provider/sockjs-provider.tsx

```diff
@@ -23,7 +24,7 @@ const SockJSProvider = ({ children }: PropsWithChildren) => {
 
   const client = useRef<CompatClient>(Stomp.over(sock));
 
-  const authToken = getLocalStorage('token');
+  const authToken = tokenStorage.getItem();
```

Comment:
다른 파일에선 token이라는 변수명으로 값을 저장하고 있는데요~
여기서도 동일하게 표현해줘도 괜찮을 것 같습니다~
```suggestion
  const token = tokenStorage.getItem();
```